jobs/JREGIONAL_GET_EXTRN_MDL_FILES

```diff
@@ -242,7 +250,7 @@ Call to ex-script corresponding to J-job \"${scrfunc_fn}\" failed."
 #
 #-----------------------------------------------------------------------
 #
-job_postamble "FALSE"
+# job_postamble
```

Comment:
There are four jobs in AQM using its DATA to put the job output file for other job to access as input. The data directory of four jobs can not be removed when KEEPDATA=NO. Therefore a dedicated data_cleanup job was developed to handle it.

Reply:
"KEEPDATA=NO" doesn't work in this workflow because "YES/NO" is not the standard flag in the UFS SRW App. If you set "KEEPDATA: false" in `config.yaml`, it should work. I've test this several times. So please do NOT change the 'KEEPDATA'-related parts in this PR.